src/de/2024-04/12/teacher-questions.md

```diff
@@ -11,71 +11,72 @@ date:   20/12/2024
 
 ## STARTFRAGEN
 
--   ...
+-   -   …
 
-## SONNTAG -- Was ist Wahrheit?
+## SONNTAG – Was ist Wahrheit?
 
-**Joh 18,33-38**
+'''Joh 18,33-38““
 
 -   Fragen zum Text:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
```

Comment:
@toak What are three dots suppose to represent?

Reply:
that is when the editor still need to put questions, is always removed in the transformation